kubernetes/samples/scripts/create-weblogic-domain-on-azure-kubernetes-service/create-domain-on-aks-inputs.sh

```diff
@@ -0,0 +1,96 @@
+# Copyright (c) 2018, 2021, Oracle and/or its affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+# The version of this inputs file. Do not modify.
+export version=create-domain-on-aks-inputs-v1
+
+#
+# Parameters that must be changed from these values!
+#
+
+# Oracle Single Sign-On (SSO) account email, used to pull the WebLogic Server image.
+export dockerEmail=docker-email
+
+# Oracle SSO account password, used to pull the WebLogic Server image.
+export dockerPassword=docker-password
+
+# Name of weblogic user account.
+export weblogicUserName=weblogic
+
+# Password for WebLogic user account.
+export weblogicAccountPassword=Secret123456
+
+# Specify where to create azure resource.
+export azureLocation=eastus
+
+# Specify a prefix to name resources, only allow lowercase letters and numbers, between 1 and 7 characters.
+# Resource group is named with ${namePrefix}resourcegroup<timestamp>, e.g. wlsresourcegroup1592469388
+# Kubernetes cluster is named with ${namePrefix}akscluster<timestamp>, e.g. wlsakscluster1592469388
+# Storage account is named with ${namePrefix}storage<timestamp>, e.g. wlsstorage1592469388
+export namePrefix=wls
+
+#
+# Parameters that may optionally be changed.
+#
+
+# The suffix of file share secret name, the complete value is ${namePrefix}${azureFileShareSecretNameSuffix}.
+export azureFileShareSecretNameSuffix=azure-secret
```

Comment:
Is this a hard coded secret?

Reply:
no, this is the name of the secret, not the secret value.